edgelet/dps/src/registration.rs

```diff
@@ -16,14 +16,14 @@ use tokio::prelude::*;
 use tokio::timer::Interval;
 use url::form_urlencoded::Serializer as UrlSerializer;
 
-use edgelet_core::crypto::{Activate, KeyIdentity, KeyStore, Sign, Signature, SignatureAlgorithm};
-use edgelet_http::client::{Client, ClientImpl, TokenSource};
-use edgelet_http::ErrorKind as HttpErrorKind;
-use error::{Error, ErrorKind};
-use model::{
+use crate::error::{Error, ErrorKind};
+use crate::model::{
     DeviceRegistration, DeviceRegistrationResult, RegistrationOperationStatus, TpmAttestation,
     TpmRegistrationResult,
 };
+use edgelet_core::crypto::{Activate, KeyIdentity, KeyStore, Sign, Signature, SignatureAlgorithm};
```

Comment:
Order of imports should be libstd (`use std::`) -> external crates (`use tokio::`) -> our crates (`use edgelet_core::`) -> current crate (`use crate::`)